LeaderboardBackend.Test/Features/Users/AccountConfirmationTests.cs

```diff
@@ -0,0 +1,96 @@
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Services;
+using LeaderboardBackend.Test.Fixtures;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test.Features.Users;
+
+[TestFixture]
+public class AccountConfirmationTests : IntegrationTestsBase
+{
+    private const string RESEND_CONFIRMATION_URI = "/account/confirm";
+    private IServiceScope _scope = null!;
+    private IAuthService _authService = null!;
+    private IUserService _userService = null!;
+
+    [OneTimeSetUp]
```

Comment:
Happy to change it, but in the meantime, but could you explain why it could be bad? I can't really see why.

Also, I'm rereading [the docs on scoped services](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection#scoped), and it warns to not resolve them from a singleton. Since we're resolving them from `s_factory`, which is a static member of a parent class, wouldn't that make it one?

Reply:
Tests could share state and affect each others unexpectedly.
For example the DbContext's change tracker is shared in one scope, so one test could save another test's entity changes when calling .SaveChanges()

Reply:
I see. I haven't considered that.